docs/excel/excel-add-ins-worksheets.md

```diff
@@ -254,52 +253,78 @@ Excel.run(function (context) {
 }).catch(errorHandlerFunction);
 ```
 
-## Get a cell within a worksheet
+## Find data in a worksheet
+
+This section describes how to find cells and ranges using the `Worksheet` object's functions of `getCell` and `findAll`. More range retrieval information can be found in object-specific articles.
+
+- For examples that show how to get a range within a worksheet using the `Range` object, see [Work with ranges using the Excel JavaScript API](excel-add-ins-ranges.md).
+- For examples that show how to get ranges from a `Table` object, see [Work with tables using the Excel JavaScript API](excel-add-ins-tables.md).
+- For examples that show how to search a large range for multiple sub-ranges based on cell characteristics, see [Work with multiple ranges simultaneously in Excel add-ins](excel-add-ins-multiple-ranges.md).
 
-The following code sample gets the cell that is located in row 2, column 5 of the worksheet named **Sample**, loads its **address** and **values** properties, and writes a message to the console. The values that are passed into the **getCell(row: number, column:number)** method are the zero-indexed row number and column number for the cell that is being retrieved.
+### Get a cell within a worksheet
+
+The following code sample gets the cell that is located in row 2, column 5 of the worksheet named **Sample**, loads its **address** and **values** properties, and writes a message to the console. The values that are passed into the `getCell(row: number, column:number)` method are the zero-indexed row number and column number for the cell that is being retrieved.
 
 ```js
 Excel.run(function (context) {
     var sheet = context.workbook.worksheets.getItem("Sample");
     var cell = sheet.getCell(1, 4);
     cell.load("address, values");
-    
+
     return context.sync()
         .then(function() {
             console.log(`The value of the cell in row 2, column 5 is "${cell.values[0][0]}" and the address of that cell is "${cell.address}"`);
         })
 }).catch(errorHandlerFunction);
 ```
 
-## Get a range within a worksheet
+### Find all ranges with matching text (preview)
```

Comment:
Suggest renaming this section -> **Find data in a worksheet (preview)** and promoting it to an H2. Reason being -- "find data in a worksheet" describes the task that the reader is likely to have in mind -- they aren't likely to think of it in terms of "find all ranges with matching text" (even though that technically describes what the API does).

`## Find data in a worksheet (preview)`